app.use(function (err, req, res, next) {
  console.error(err.stack)
  next(err)
})


app.use(function (err, req, res, next) {
  res.status(500)
  res.render('error', { error: err })
})